cmd/containerd/main.go

```diff
@@ -168,14 +169,18 @@ func before(context *cli.Context) error {
 		}
 		fh.Close()
 		log.G(global).Infof("containerd default config written to %q", fh.Name())
-		os.Exit(0)
+		return nil
```

Comment:
I get it now, `containerd --config default` was supposed to write default config to a tmp file and exit, it doesn't run containerd at all, it looks odd to me though.

I've updated and removed that line of change.